src/main/java/org/example/coursesService/Course.java

```diff
@@ -0,0 +1,56 @@
+package org.example.coursesService;
+
+import java.util.List;
+
+public class Course {
+    private String name;
+    private String id;
+    private static List<String> enrolledStudents;
```

Comment:
This field should not be static. If you had multiple course objects the `enrolledStudents` would always be pointing to the same list